### ISSUE TITLE
Cannot go full screen on iPad

### ISSUE THREAD

**Issue by niosus** [lumetrium/obsidian-teleprompter#21]
Hello again! I seem to have lost an ability to change the size of the teleprompter panel when using it on my iPad. I could do it some time ago but since a couple of months this does not work anymore. Any ideas what might have changed? 

When I click on the "fullscreen" button it toggles but does nothing. Is this maybe a know issue with a simple workaround?

**Comment by lumetrium** [comment]
Hi! Thanks for reporting this. I made some changes to the logic behind the fullscreen control in the last update. I wasn't aware that it broke on iPads. I tested it on my Android devices, and everything worked fine. I'll take a look now and see what might be causing the issue. Not sure yet how to test it though, I should probably look into iOS/iPadOS simulators.

**Comment by lumetrium** [comment]
This is going to be difficult for me to test the plugin on Apple mobile devices. I don't own any, and all the emulators I could find are either very expensive, require a Mac, or straight up don't work.

I've reverted the previous changes to the fullscreen control for the iOS app. Could you please update the plugin to the latest version 1.4.2 that I just published and try going fullscreen again?

https://github.com/lumetrium/obsidian-teleprompter/releases/tag/1.4.2

**Comment by niosus** [comment]
Thanks for your help and sorry for the delay. I updated the plugin today but still have the same issue. Is there anything else I can do to help you debug this?

Just for completeness I am attaching the video of what it does (or rather doesn't do)

https://github.com/user-attachments/assets/ff9fe2be-99b6-4106-94c1-d3377cda422d

**Comment by lumetrium** [comment]
Interesting. Are you sure you used the fullscreen feature before and it worked? 

I need to see what errors, if any, are present in the console. Unfortunately, accessing it on iOS isn't exactly straightforward. According to the [Obsidian docs](https://docs.obsidian.md/Plugins/Getting+started/Mobile+development#iOS):

> You can inspect Obsidian on an iOS device running 16.4 or later and a macOS-based computer. Instructions on how to set it up can be found here: [https://webkit.org/web-inspector/enabling-web-inspector/](https://webkit.org/web-inspector/enabling-web-inspector/)

Do you happen to own a Mac? If so, would you be able to follow the instructions from the WebKit docs to enable remote inspection on an iOS device? Then, open the console, try to activate the fullscreen mode on your iPad again, and send me a screenshot of the console or copy-paste its contents.

**Comment by niosus** [comment]
I'm sure that I could run it before, see my other issue, where I show a video of how it looked before #16

I have a Mac, so I should be able to see the logs and send them your way. 👍

**Comment by niosus** [comment]
Although, looking at it again, it might be that I never used the fullscreen option and managed to change the size of the panel somehow 🤔 The button doesn't look to be pressed on that video.

**Comment by lumetrium** [comment]
Yeah, I noticed that too. Chances are the fullscreen feature never worked on iPads. I'll be waiting the console logs from you so I can try to fix it.

**Comment by RubenTarne** [comment]
Hello! I have the same issue on my iPhone (14 pro max). When I click the fullscreen option but it does not do anything. On my macbook pro (m1) it works like a charm.
